cli/lib/kontena/cli/helpers/exec_helper.rb

```diff
@@ -0,0 +1,32 @@
+module Kontena::Cli::Helpers
+  module ExecHelper
+
+    # @param [WebSocket::Client::Simple] ws 
+    # @return [Thread]
+    def stream_stdin_to_ws(ws)
```

Comment:
If we want complete support for interactive TTY execs, then we may also need to consider support for things like `SIGTTYWINCH` to send Docker API `POST exec/.../resize?h=40&w=80`?